homeassistant/components/renson/__init__.py

```diff
@@ -56,3 +63,66 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+def setup_hass_services(hass: HomeAssistant, renson_api: RensonVentilation) -> None:
+    """Set up the Renson platforms."""
+
+    async def set_timer_level(call: ServiceCall) -> None:
+        """Set timer level."""
+        level_string = call.data.get("timer_level")
+        time = call.data.get("time")
+        level = Level[str(level_string).upper()]
+
+        await hass.async_add_executor_job(renson_api.set_timer_level, level, time)
+
+    async def set_breeze(call: ServiceCall) -> None:
+        """Configure breeze feature."""
+        level = call.data.get("breeze_level")
+        temperature = call.data.get("temperature")
+        activated = call.data.get("activate")
+
+        await hass.async_add_executor_job(
+            renson_api.set_breeze, level, temperature, activated
+        )
+
+    async def set_day_night_time(call: ServiceCall) -> None:
+        """Configure day night times."""
+        day = call.data.get("day")
+        night = call.data.get("night")
+
+        await hass.async_add_executor_job(renson_api.set_time, day, night)
+
+    async def set_pollution_settings(call: ServiceCall) -> None:
+        """Configure pollutions settings."""
+        day = call.data.get("day_pollution_level")
+        night = call.data.get("night_pollution_level")
+        humidity_control = call.data.get("humidity_control")
+        airquality_control = call.data.get("airquality_control")
+        co2_control = call.data.get("co2_control")
+        co2_threshold = call.data.get("co2_threshold")
+        co2_hysteresis = call.data.get("co2_hysteresis")
```

Comment:
Why did you remove the default value?

`.get` without a default value will return `None` if the key was not present.
As example the library is expecting a `bool` for `airquality_control`.
If `None` is passed for it, the following error will occur: 
```
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

Therefore I suggest setting the correct default values.

```suggestion
        humidity_control = call.data.get("humidity_control", False)
        airquality_control = call.data.get("airquality_control", False)
        co2_control = call.data.get("co2_control", False)
        co2_threshold = call.data.get("co2_threshold", 0)
        co2_hysteresis = call.data.get("co2_hysteresis", 0)
```